docs/make.jl

```diff
@@ -0,0 +1,25 @@
+using Documenter, Convex
+
+makedocs(;
+    modules = [Convex],
+    format = Documenter.HTML(),
+    pages = [
+        "Home" => "index.md",
+        "Installation" => "installation.md",
+        "Quick Tutorial" => "quick_tutorial.md",
+        "Basic Types" => "types.md",
+        "Supported Operations" => "operations.md",
+        "Examples" => "examples.md",
+        "Complex-domain Optimization" => "complex-domain_optimization.md",
+        "Solvers" => "solvers.md",
+        "FAQ" => "faq.md",
+        "Optimizing in a Loop" => "loop.md",
+        "Advanced" => "advanced.md",
+        "Contributing" => "contributing.md",
+        "Credits" => "credits.md"
+    ],
+    repo = "https://github.com/JuliaOpt/Convex.jl/blob/{commit}{path}#L{line}",
```

Comment:
I haven't seen this before. What is it for? I think the line number part is confusing me.

Reply:
I think it’s for the “edit this on github” links on the generated docs